plugin.rb

```diff
@@ -17,13 +22,17 @@
 PLUGIN_NAME ||= "communitarian"
 
 load File.expand_path("lib/communitarian/engine.rb", __dir__)
+load File.expand_path("lib/communitarian/stripe.rb", __dir__)
 
 after_initialize do
   [
     "../app/models/communitarian/post_delay",
     "../app/models/communitarian/resolution"
   ].each { |path| require File.expand_path(path, __FILE__) }
 
+  Stripe.api_key = SiteSetting.communitarian_stripe_secret_key
```

Comment:
Do I need to reload plugin/site when I update stripe key?

Reply:
@DmitryBarskov yes, we will need to rebuild the app on dicitalocean to refresh stripe key